bio/dada2/learn-errors/environment.yaml

```
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bioconductor-dada2 == 1.16.0
```

Unless a fixed patch version is necessary for a particular reason, I would unpin it so user get the latest security patches and bug fixes whenever they newly create an environment. I.e.:
```suggestion
  - bioconductor-dada2 == 1.16
```

---

Thank you for your nice words and your review @dlaehnemann ! The trailing 0 wasn't necesary, I will have to modify it in the others PR too.